### PR TITLE
2024 Summer Olympics Dashboard

### PR DESCRIPTION
**Pull request by FaithAkinyiOuma-Tec**
Hi @MasilamaniChidambaram,
I have added the following resources:

1. README file
2. Datasets zip
3. Snapshot
4. 2024 Summer Olympics Dashboard bbix file.

### CODE REVIEW

File: Sports/2024 Summer Olympics Dashboard/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# 2024 Summer Olympics Dashboard
+
+## Overview
+This project features an interactive and detailed Bold BI dashboard focusing on the 2024 Summer   Olympics, held in August 2024 in Paris. The dashboard provides insights into various aspects of the event, including medal tallies, athlete performances, country comparisons, and sports medal distribution.
```
Comment (MasilamaniChidambaram) [inline Sports/2024 Summer Olympics Dashboard/README.md]: space between  2024 Summer   Olympics

File: Sports/2024 Summer Olympics Dashboard/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Features
+- **Medal Tally**: Distribution of gold, silver, and bronze medals by country and sport.
+- **Country Comparisons**: Visual comparisons of countries’ performances across different medals.
+- **Athlete Performance**: Detailed insights into athlete achievements by gender.
+- **Interactive Visualizations**: User-friendly visuals and filters for custom views.
+
+## Files Included
+- [`Olympic 2024_Dashboard.bbix`](https://github.com/boldbi/dashboard-examples/blob/sports_dashboard_624370/Sports/2024%20Summer%20Olympics%20Dashboard/2024%20Summer%20Olympics%20Dashboard.bbix): The Bold BI dashboard file.
```
Comment (MasilamaniChidambaram) [inline Sports/2024 Summer Olympics Dashboard/README.md]: this should be from master branch, not from your branch -https://github.com/boldbi/dashboard-examples/blob/master/Sports/2024%20Summer%20Olympics%20Dashboard/2024%20Summer%20Olympics%20Dashboard.bbix

File: Sports/2024 Summer Olympics Dashboard/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+- `Data/`: Folder containing raw data files used for the dashboard.
+- `README.md`: This documentation file.
+
+## Installation
```
Comment (MasilamaniChidambaram) [inline Sports/2024 Summer Olympics Dashboard/README.md]: Belowv3 steps are not clear to use dashboard

File: Sports/2024 Summer Olympics Dashboard/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+## Data Sources
+- **Medals**: Official data provided by the International Olympic Committee.
+- **Medal_Lists**: Supplementary data for medals distribution statistics and country profiles.
+
+## Usage
+- **Filtering**: Use the provided filters to customize views based on country, sport, event, and athlete.
+- **Export**: Export visualizations and data tables as needed.
+
+## Contributing
```
Comment (MasilamaniChidambaram) [inline Sports/2024 Summer Olympics Dashboard/README.md]: Update proper read me file

**Review by MasilamaniChidambaram — COMMENTED** [review]
dataset, readme file, snapshot should be inside of dashboard folder.

File: Sports/2024 Summer Olympics Dashboard/README.md (hunk 4 above)
Comment (FaithAkinyiOuma-Tec) [reply Sports/2024 Summer Olympics Dashboard/README.md]: Hi @MasilamaniChidambaram,
I have addressed the mentioned concerns. Kindly check and let me know your further doubts.
https://github.com/boldbi/dashboard-examples/commit/4bf08abb6782bbc63ba798d7cad47e8372e1f7f5

Regards,
Faith.

**Pull request closed without merging**